src/app/pages/update-profile-request/components/profile-box.tsx

```diff
@@ -0,0 +1,73 @@
+import { Profile } from '@stacks/profile';
+import { color, Stack } from '@stacks/ui';
+
+function Value({ v }: { v: any }) {
+  if (typeof v === 'string' || typeof v === 'number' || typeof v === 'boolean') {
+    return <span style={{ wordBreak: 'break-all' }}>{v}</span>;
+  } else if (v instanceof Array) {
+    return (
+      <>
+        [
+        <div style={{ textIndent: '0em' }}>
+          {v.map((entry, index) => {
+            return (
+              <div key={index}>
+                <Value v={entry} />
+                {index < v.length - 1 && <>,</>}
+              </div>
+            );
+          })}
+        </div>
+        ]
+      </>
+    );
+  } else if (typeof v === 'object') {
+    return <Properties p={v} />;
+  } else {
+    return <>{JSON.stringify(v)}</>;
+  }
+}
+
+function Properties({ p }: { p: any }) {
+  return (
+    <>
+      {Object.keys(p)
+        .filter(k => !k.startsWith('@'))
+        .map(k => {
+          return (
+            <div key={k} style={{ textIndent: '-1em', marginLeft: '1em' }}>
```

Comment:
Can you make use of our UI library elements here and throughout?

Reply:
Are there any guidelines? I am not sure what component would be appropriate here.

Reply:
@kyranjamie 

Before:
![image](https://user-images.githubusercontent.com/1449049/200652805-f1ff62f3-29e7-496f-9c81-02efa6dbbd75.png)

After:
![image](https://user-images.githubusercontent.com/1449049/200653127-9de186a6-4004-4847-b87e-f0cc6bc36cbe.png)